core/conn/unixodbc/odbc/odbcclient/unixcli/TCPIPV4/TCPIPV4.RC

```diff
@@ -45,11 +45,11 @@ BEGIN
         BLOCK "000004b0"
         BEGIN
             VALUE "Comments", "T7968G10_01NOV2004_ABC_IPV4_1220\0"
-            VALUE "CompanyName", "Hewlett-Packard Company\0"
+            VALUE "CompanyName", "Esgyn Corporation\0"
             VALUE "FileDescription", "NonStop� ODBC/MX TCPIPV4 DLL\0"
```

Comment:
I'm wondering what the significance of "CompanyName" is. Perhaps we should put Apache Software Foundation here? Also, I'm wondering if we should take out "NonStop ODBC/MX" and replace it with something like "Apache Trafodion ODBC"?